### PR TITLE
Create Spiral Entry

### LINKED ISSUE #1808: [Term Entry] General SDLC: Spiral
### Reason/inspiration (optional)

The [Software Development Lifecycle](https://www.codecademy.com/resources/docs/general/software-development-life-cycle) Model to cover:

- Spiral: Based on the waterfall and iterative models.

Please write a new entry and expand on the above introductory paragraph under:  content/general/concepts/software-development-life-cycle/terms/ about the Spiral model.

To get started with the PR and entry file:

Under content/general/concepts/software-development-life-cycle/terms/, create a new folder called `spiral`. Then create a new file called `spiral.md` under content/general/concepts/software-development-life-cycle/terms/`spiral`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by hongquant17**
### Description

Added new folder `spiral` inside content/general/concepts/software-development-life-cycle/terms/.
Added new entry file `content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md` for General SDLC: Spiral.

Closes #1808

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+Title: "Spiral"
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:2]:
```suggestion
Title: 'Spiral'
```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Description: 'A model of software development which is a combination of Waterfall model and Iterative development.'
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:3]:
```suggestion
Description: 'A model of software development that combines the Waterfall model with iterative development.'
```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The **Spiral** model is a Software Development Life Cycle (SDLC) which is an excellent combination between Waterfall model and Iterative development. For each iteration, phases are repeatedly developed, with Risk Analysis and Prototype integrated.
+
+The Spiral model was first introduced by Barry Boehm in 1986 as a result of applying iterative development in constructing softwares.
+
+The workflow is divided in four phases:
+- Planning
+- Risk Analysis
+- Engineering
+- Evaluation
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:23]:
```suggestion
1. Planning
2. Risk Analysis
3. Engineering
4. Evaluation
```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Each iteration consists of all above phases but has different aims which are usually:
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:25]:
```suggestion
Each iteration consists of all above phases but has different aims which are usually:

```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- Prototype
+- Release candidate
+- Launch
+
+## Advantages:
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:30]:
```suggestion
## Advantages:

```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- Products are soon introduced with prototype.
+- Errors and failures are identified and handled at ease.
+- Frequent feedback and refinement through cycles.
+- Flexible, adaptive to changes.
+
+## Disadvantages:
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:36]:
```suggestion
## Disadvantages:

```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+- Complex workflow and required expertise especially in Risk Analysis.
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:37]:
```suggestion
- Complex workflow and requires expertise especially in risk analysis.
```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+  - 'Web Development'
+Tags:
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **Spiral** model is a Software Development Life Cycle (SDLC) which is an excellent combination between Waterfall model and Iterative development. For each iteration, phases are repeatedly developed, with Risk Analysis and Prototype integrated.
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:15]:
```suggestion
The **Spiral model** is a Software Development Life Cycle (SDLC) that combines the [Waterfall model](https://www.codecademy.com/resources/docs/general/software-development-life-cycle/waterfall) and iterative development approaches. Each phase within this model involve risk analysis and prototyping.
```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+The Spiral model was first introduced by Barry Boehm in 1986 as a result of applying iterative development in constructing softwares.
+
+The workflow is divided in four phases:
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:19]:
```suggestion
The Spiral model was first introduced by Barry Boehm in 1986 as a result of applying iterative development in constructing softwares. It is divided into four phases:
```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+Each iteration consists of all above phases but has different aims which are usually:
+- Prototype
+- Release candidate
+- Launch
+
+## Advantages:
+- Products are soon introduced with prototype.
+- Errors and failures are identified and handled at ease.
+- Frequent feedback and refinement through cycles.
+- Flexible, adaptive to changes.
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:34]:
```suggestion
- Products are prototyped for quick introduction.
- Errors and failures are identified and handled as they arise.
- Frequent feedback and refinement through cycles.
- Flexible and adaptive to changes.
```

**Review by yangc95 — COMMENTED** [review]
@yoiang Thanks for your patience!

I left a few comments and suggestions. Otherwise, nice and simple entry we have here 👍

**Comment by hongquant17** [comment]
@yangc95 Thanks for your comments, I have modified the entry according to your suggestions. Hope to receive the your next feedback soon

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **Spiral model** is a Software Development Life Cycle (SDLC) that combines the [Waterfall model](https://www.codecademy.com/resources/docs/general/software-development-life-cycle/waterfall) and iterative development approaches. Each phase within this model involve risk analysis and prototyping.
+
+The Spiral model was first introduced by Barry Boehm in 1986 as a result of applying iterative development in constructing softwares. It is divided into four phases:
```
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:17]:
```suggestion
The Spiral model was first introduced by Barry Boehm in 1986 as a result of applying iterative development in constructing softwares. It is divided into four phases:

```

**Review by yangc95 — APPROVED** [review]
@hongquant17 Looks good! Thanks for incorporating those suggestions. Marking this as ready for another review 😸 

(Just make sure to add that new line suggestion I made)

**Comment by hongquant17** [comment]
@yangc95  Thanks for your review, new line suggestion was resolved

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+  - 'Web Development'
+Tags:
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **Spiral model** is a Software Development Life Cycle (SDLC) that combines the [Waterfall model](https://www.codecademy.com/resources/docs/general/software-development-life-cycle/waterfall) and iterative development approaches. Each phase within this model involve risk analysis and prototyping.
```
Comment (SSwiniarski) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:15]:
```suggestion
The **Spiral model** is a Software Development Life Cycle (SDLC) that combines the [Waterfall model](https://www.codecademy.com/resources/docs/general/software-development-life-cycle/waterfall) and iterative development approaches. Each phase within this model involves risk analysis and prototyping.
```

File: content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+The Spiral model was first introduced by Barry Boehm in 1986 as a result of applying iterative development in constructing softwares. It is divided into four phases:
```
Comment (SSwiniarski) [inline content/general/concepts/software-development-life-cycle/terms/spiral/spiral.md:17]:
```suggestion
The Spiral model was first introduced by Barry Boehm in 1986 as a result of applying iterative development in constructing software. It is divided into four phases:
```

**Review by SSwiniarski — COMMENTED** [review]
@hongquant17 I did a second review and left a couple of comments.

**Comment by hongquant17** [comment]
@SSwiniarski thank you for your review, I have added your suggestions to the entry. Looking forward to your next feedback.

**Review by SSwiniarski — APPROVED** [review]
@hongquant17 Looks good!

**Comment by SSwiniarski** [comment]
@hongquant17 Congrats on the new entry. Live link is here: https://www.codecademy.com/resources/docs/general/software-development-life-cycle/spiral

image

**Comment by hongquant17** [comment]
@SSwiniarski @yangc95 thank you both for your support!

**Pull request merged**